src/OrbitGl/App.cpp

```diff
@@ -1743,7 +1743,11 @@ void OrbitApp::AddFrameTrack(uint64_t instrumented_function_id) {
 }
 
 void OrbitApp::RemoveFrameTrack(const FunctionInfo& function) {
-  CHECK(HasCaptureData());
```

Comment:
Sorry for being later. I'd argue to keep the CHECK and move the if to the caller. RemoveFrameTrack should remove an existing frametrack from the ui. It does not make sense to call it if there is not CaptureData, thus no existing frametrack.

Reply:
true - I will do it in main branch... I would keep it as is here

Reply:
Actually it is used in 5 different places. I think it is better if we just ignore this here. Note that this call is also ignored if the function was not previously instrumented.